tools/python/blas/check_cblas.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf8 -*-
+
+# python setup.py build_ext --inplace
+
+"""
+setup.py file
```

Comment:
Maybe can we explain here that this file is here to see if we can link against blas

Reply:
sure